src/cocaine-app/sched/ttl_cleanup_starter.py

```diff
@@ -0,0 +1,119 @@
+import logging
+
+from infrastructure import infrastructure
+import jobs
+from mastermind_core.config import config
+import storage
+import time
+import datetime
+from yt_worker import YqlWrapper
+
+
+logger = logging.getLogger('mm.sched.ttl_cleanup')
+
```

Comment:
fixes in https://github.com/yandex/mastermind/pull/434/commits/5fbbb5ad5a36a25226ac702b5962feac4660e282